packages/deployments/contracts/contracts/core/connext/helpers/OZERC20.sol

```diff
@@ -2,16 +2,15 @@
 pragma solidity 0.8.17;
 
 // This is modified from "@openzeppelin/contracts/token/ERC20/IERC20.sol"
-// Modifications were made to make the tokenName, tokenSymbol, and
-// tokenDecimals fields internal instead of private. Getters for them were
-// removed to silence solidity inheritance issues
+// Modifications were made to allow the name, hashed name, and cached
+// domain separator to be internal
 
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/extensions/draft-IERC20Permit.sol";
-import {EIP712} from "@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol";
+import {EIP712} from "@openzeppelin/contracts/utils/cryptography/EIP712.sol";
```

Comment:
> Can use https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/extensions/ERC20Permit.sol instead of `draft-IERC20Permit.sol`

https://github.com/OpenZeppelin/openzeppelin-contracts/tree/49c0e4370d0cc50ea6090709e3835a3091e33ee2

OZ latest release is 4.8.0 and it doesn't include `ERC20Permit.sol`.

Reply:
Oh, I see, the version with `ERC20Permit.sol` isn't release yet (although it is in the main branche)
No problem to leave it.